datadog/resource_datadog_integration_pagerduty_service_object.go

```diff
@@ -82,41 +84,46 @@ func resourceDatadogIntegrationPagerdutySORead(d *schema.ResourceData, meta inte
 }
 
 func resourceDatadogIntegrationPagerdutySOExists(d *schema.ResourceData, meta interface{}) (b bool, e error) {
-	client := meta.(*datadog.Client)
+	providerConf := meta.(*ProviderConfiguration)
+	client := providerConf.CommunityClient
 
 	_, err := client.GetIntegrationPDService(d.Id())
 	if err != nil {
 		if strings.Contains(err.Error(), "404 Not Found") {
 			return false, nil
 		}
-		return false, err
+		return false, translateClientError(err, "error getting pager duty integration service")
```

Comment:
Nit, these should stay PagerDuty

Reply:
Another nit, ,should we also say error checking exists to keep it consistent, like - https://github.com/terraform-providers/terraform-provider-datadog/pull/450/files#diff-f51916c63534715f41c1d444931ae717R370